example/spec/parallel/ingestGranule/IngestGranuleFailureSpec.js

```diff
@@ -214,8 +214,8 @@ describe('The Ingest Granule failure workflow', () => {
       const granule = JSON.parse(granuleResponse.body);
 
       expect(granule.status).toBe('failed');
-      expect(granule.error.Error).toBe(syncGranFailedDetail.error);
-      expect(JSON.parse(granule.error.Cause)).toEqual(JSON.parse(syncGranFailedDetail.cause));
+      expect(granule.error.Error).toBeDefined();
```

Comment:
The title of this test should be updated to be more ... exact about what we're testing.